INSTALL.md

```diff
@@ -15,16 +15,23 @@ run:
 ## Prerequisites
 
 All platforms require the `automake` and `libtool` suite of tools installed in
-order to build Vere. Before going any further, install them using your package
-manager. For example, on macOS:
+order to build Vere.  On Linux, `bazel` also needs to be configured before Vere can be installed.
+
+### MacOS
+
+Install `automake` and `libtool` using [`homebrew`][homebrew]:
 
 ```console
 $ brew install automake libtool
 ```
 
+Now MacOS should be ready to use `bazel` to build Vere.
+
 ### Linux
 
-We use [musl libc][musl libc] instead of [glibc][glibc] on Linux, which enables us
+Make sure `automake` and `libtool` are installed on your system.  TODO: describe how to install these on common distros
```

Comment:
I think moving instructions to install `automake` and `libtool` under OS-specific headers adds volume without adding value. Instead, I'd suggest the following:

```
## Prerequisites
<no change>

### Linux

After installing `automake` and `libtool`, you need to install the [musl libc] toolchain. We use [musl libc][musl libc] instead of [glibc][glibc] on Linux, which enables us to generate statically linked binaries.

#### x86_64

<snip>

### macOS
After installing `automake` and `libtool`, you're ready to build Vere.
```

Reply:
Also, as @matthew-levan pointed out, `MacOS` should be `macOS`.